examples/polygon/run.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
There are three `run.sh` files in this PR. Is there a benefit to combining them into a single parametrized script?